examples/jsm/interactive/HTMLMesh.js

```diff
@@ -273,6 +273,12 @@ function html2canvas( element ) {
 			context.drawImage( element, 0, 0 );
 			context.restore();
 
+		} else if ( element instanceof HTMLImageElement ) {
+
+			if ( element.style.display === 'none' ) return;
+
+			context.drawImage( element, 0, 0, element.width, element.height );
```

Comment:
Shouldn't this be `buildRectPath( x, y, width, height, 2 ); context.drawImage( element, x, y, element.width, element.height );`, as in https://github.com/mrdoob/three.js/pull/24043?

Reply:
I think it's less confusing to just draw the image and not an additional path.

Reply:
Having `0, 0` here will always draw the image starting on the left top corner. That's definitely not my use case.
I'm using a variant of the above, this:
```js
		} else if ( element instanceof HTMLImageElement ) {

			if ( element.style.display === 'none' ) return;

			const rect = element.getBoundingClientRect();

			x = rect.left - offset.left - 0.5;
			y = rect.top - offset.top - 0.5;
			width = rect.width + 0.5;
			height = rect.height + 0.5;
			
			context.drawImage( element, x, y, width, height );
		} else {
```

It works great for my use case of having a wrist UI to change the texture of a furniture.
Here is the screenshot from chrome, but that's exactly the same render in VR:
![Capture d’écran du 2023-04-24 16-55-47](https://user-images.githubusercontent.com/112249/234035279-45cd7658-2d8b-42dd-90f1-8ae88843f716.png)

Do you want me to create a PR with those changes?

cc @msub2

Reply:
PR created https://github.com/mrdoob/three.js/pull/25916